src/components/pages/search.js

```diff
@@ -33,9 +33,9 @@ function Search({ PageTemplate }) {
   const [selectedTags, setSelectedTags] = useState([]);
   const [displayedRows, setDisplayedRows] = useState([]);
   const [unselectedTags, setUnselectedTags] = useState([]);
-  const [currentPagination, setCurrentPagination] = useState(1);
   const [searchText, setSearchText] = useState(request.query.query);
   const [sorting, setSorting] = useState({ value: 'relevance', direction: 'desc' });
+  const [currentTablePagination, setCurrentTablePagination] = useState({ current: 1, pageSize: 10, showSizeChanger: true });
```

Comment:
It looked a bit buggy that ant was not always showing the size changer (showing it only with a data set of more than 50 rows), so I enabled it all the time.

Reply:
You need to actively look into this in order to find that out :roll_eyes: